backend/tests/viewAppointmentsController.test.ts

```diff
@@ -383,7 +391,7 @@ describe('View appointments', () => {
                 attributes: { 'data-sort-value': 'FOUR' },
                 html: '<a href="/prisoner/ABC456" class="govuk-link">Four, Offender - ABC456</a>',
               },
-              { text: undefined },
+              { text: '2-1-1' },
```

Comment:
since we've called at the start and then read from the map we have results for this prisoner.  Previous mocking only mocked the first three calls, hence last 3 were undefined.